showcase/src/app/demo/button/button.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit, Inject, ViewEncapsulation } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+@Component({
+	selector: "showcase-demo-button",
+	templateUrl: "./button.component.html",
+	styleUrls: ["./button.component.scss"],
+	encapsulation: ViewEncapsulation.None
+})
+export class ButtonComponent implements OnInit {
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService) {}
+
+	public ngOnInit(): void {
+		console.log("On Init");
```

Comment:
Replace the console.log by the StarkLoggingService since you already injected it ;)

Reply:
oups my mistake :-)